### PR TITLE
Added NeptuneMC Network

### PR DESCRIPTION
**Pull request by kinngzzz**
### General:
* [x] The pull request title is descriptive. *(ex. `Added Lunar Network`, not `Updated servers.json`)*
* [x] The pull request does not contain any unrelated commits.  *(ex. commits from previous pull requests)*

### Mapping Additions or Updates:
* [x] No changes were made to the file's formatting.
* [x] There are no syntax errors.
* [x] There is no pre-existing mapping matching my id.
* [x] My field values match your requirements.
* You can view our patterns here: [servers.schema.json](https://github.com/LunarClient/ServerMappings/blob/master/servers.schema.json), or take a look below and complete the field checklist:
  - [x] `id`: a lowercase string *(ex. `myserver`)*
  - [x] `name`: a string *(ex. `MyServerPvP`)*
  - [x] `addresses`: an array with lowercase strings *(ex. of address `my.server`)*
    - You do not need to specify sub-domains, Lunar Client services automatically detect them.

### Logo Additions or Updates:
* [x] My image is a `png` file.
* [x] I uploaded my image to the `logos` directory.
* [x] My image has a transparent background and is square (1:1 aspect ratio).
* [x] My image is `512` pixels in width and height.
* [x] My image's file name matches its mapping id.

### CODE REVIEW

**Comment by thekingdmb** [comment]
PNG on 512x51
neptunemc512
2

**Review by SrIzan10 — CHANGES_REQUESTED** [review]
and... the ip?

File: servers.json [hunk 1]
```diff
@@ -1522,5 +1522,12 @@
         "addresses": [
             "revenge.cc"
         ]
+    },
+    {
+        "id": "neptunemc",
+        "name": "NeptuneMC Network",
+        "addresses": [
+            "neptunemc"
```
Comment (GrowlyX) [inline servers.json:1530]: This is an invalid address.

**Review by GrowlyX — CHANGES_REQUESTED** [review]

File: servers.json (hunk 1 above)
Comment (SrIzan10) [reply servers.json:1530]: yeah i agree

Comment (thekingdmb) [reply servers.json:1530]: Oops, srry it's neptunemc.es

Comment (thekingdmb) [reply servers.json]: Updated tho

**Review by SrIzan10 — DISMISSED** [review]
fair enough

**Review by itsJordan — CHANGES_REQUESTED** [review]
Please include the following properties into your servers.json:

primaryAddress
minecraftVersions
primaryMinecraftVersion

Consult the updated README for more information.

**Comment by thekingdmb** [comment]
Added on the server.json
primaryAddress
minecraftVersions
primaryMinecraftVersion

**Review by itsJordan — CHANGES_REQUESTED** [review]
Please adjust your 'minecraftVersions' field to match the available options correctly:
"1.7", "1.8", "1.12", "1.16", "1.17", "1.18", "1.19"

**Review by itsJordan — APPROVED** [review]

**Review by itsJordan — APPROVED** [review]

**Pull request merged**